lib/src/dartdoc_options.dart

```diff
@@ -450,7 +450,7 @@ abstract class DartdocOption<T> {
 
   /// To avoid accessing early, call [add] on the option's parent before
   /// looking up unless this is a [DartdocRootOption].
-  late final DartdocOption parent;
+  late final DartdocOption<dynamic> parent;
```

Comment:
It was not necessary; it is a no-op. I was just trying to avoid _implicit_ dynamic. Should I revert?

Reply:
Oh.  I actually don't know what is better here, I suppose I'm against putting dynamic in just in case some magic in the type system allows for something more specific later.   That isn't really on solid ground though?

Reply:
Right, type args in a declaration are never filled in by inference. The `strict-raw-types` analysis mode would catch each such case, so that the human doesn't have to guess at where inference happens and where it doesn't.

I can revert for now and add that analysis mode in a different CL?

Reply:
sounds fine